docs/apache-airflow-providers-amazon/operators/emr/emr_serverless.rst

```diff
@@ -61,8 +61,21 @@ start an EMR Serverless Job.
    :start-after: [START howto_operator_emr_serverless_start_job]
    :end-before: [END howto_operator_emr_serverless_start_job]
 
-.. _howto/operator:EmrServerlessDeleteApplicationOperator:
+.. _howto/operator:EmrServerlessStopApplicationOperator:
+
+Stop an EMR Serverless Application
+==================================
+
+You can use :class:`~airflow.providers.amazon.aws.operators.emr.EmrServerlessStopApplicationOperator` to
+stop an EMR Serverless Application.
+
+.. exampleinclude:: /../../tests/system/providers/amazon/aws/example_emr_serverless.py
+   :language: python
+   :dedent: 4
+   :start-after: [START howto_operator_emr_serverless_stop_application]
+   :end-before: [END howto_operator_emr_serverless_stop_application]
 
+.. _howto/operator:EmrServerlessDeleteApplicationOperator:
```

Comment:
```suggestion
.. _howto/operator:EmrServerlessDeleteApplicationOperator:

```

Reply:
Build docs is failing (see tests below). I think by applying the suggested change it should fix it. In any case, you can test the doc generation locally by running the command `breeze build-docs --package-filter apache-airflow-providers-amazon`